config_example.sh

```diff
@@ -319,3 +319,11 @@ set -x
 
 # Set whether static IPs will be used for all nodes or only Node0
 # export AGENT_STATIC_IP_NODE0_ONLY="true"
+
+# Set a single config variable to create a cluster for the different scenarios 
+# i.e. Single Node Openshift(SNO), Highly Available (HA) or Compact cluster.
+# The only supported values for AGENT_CONFIG are COMPACT_IPV4, COMPACT_IPV6, HA_IPV4, HA_IPV6, SNO_IPV4 and SNO_IPV6.
+# When set, the code internally sets other low level details such as disk size, memory, number of masters and workers,
+# cpu and ip stack.
+# This config variable is used only by the agent based installer.
+# export AGENT_CONFIG=SNO_IPV4
```

Comment:
Should there be a default value for AGENT_CONFIG?

Reply:
Agree, there should be an empty default in common.sh

Reply:
Done